likecoin/js/admin-settings/src/pages/LikecoinButtonPage.js

```diff
@@ -112,24 +117,23 @@ function LikecoinButtonPage() {
     e.preventDefault();
     setShowChangeButton(false);
     const data = {
-      userLikerInfos: {
-        likecoin_id:
-          likerDisplayName === '-' ? DBUserLikerId : likerIdValue,
-        display_name:
-          likerDisplayName === '-'
-            ? DBUserLikerDisplayName
-            : likerDisplayName,
-        wallet:
-          likerDisplayName === '-'
-            ? DBUserLikerWallet
-            : likerWalletAddress,
-        avatar:
-          likerDisplayName === '-'
-            ? DBUserLikerAvatar
-            : likerAvatar,
-      },
+      userLikerInfos: {},
     };
+    if (likerIdValue) setForceDisconnect(false);
+    if (forceDisconnect) {
+      data.userLikerInfos.likecoin_id = '';
+      data.userLikerInfos.display_name = '';
+      data.userLikerInfos.wallet = '';
+      data.userLikerInfos.avatar = '';
+    } else {
+      data.userLikerInfos.likecoin_id = likerDisplayName === '-' ? DBUserLikerId : likerIdValue;
+      data.userLikerInfos.display_name = likerDisplayName === '-' ? DBUserLikerDisplayName : likerDisplayName;
+      data.userLikerInfos.wallet = likerDisplayName === '-' ? DBUserLikerWallet : likerWalletAddress;
+      data.userLikerInfos.avatar = likerDisplayName === '-' ? DBUserLikerAvatar : likerAvatar;
+    }
     try {
+      // Change notice
+      if (!data.userLikerInfos.likecoin_id) setForceDisconnect(false); // has insert liker ID
```

Comment:
Why `likercoin_id` not exists means `has insert liker ID`?

Reply:
Changed it in the simplify UI logic commit.